src/main/java/de/sldk/mc/metrics/GarbageCollectorWrapper.java

```diff
@@ -0,0 +1,36 @@
+package de.sldk.mc.metrics;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.bukkit.plugin.Plugin;
+
+import io.prometheus.client.Collector;
+import io.prometheus.client.Collector.MetricFamilySamples.Sample;
+import io.prometheus.client.hotspot.GarbageCollectorExports;
+
+public class GarbageCollectorWrapper extends Metric {
+    public GarbageCollectorWrapper(Plugin plugin) {
+        super(plugin, new GarbageCollectorExportsCollector());
+    }
+
+    @Override
+    protected void doCollect() {}
+
+    private static class GarbageCollectorExportsCollector extends Collector {
+        private static final GarbageCollectorExports garbageCollectorExports = new GarbageCollectorExports();
+
+        @Override
+        public List<MetricFamilySamples> collect() {
+            MetricFamilySamples collected = garbageCollectorExports.collect().get(0);
```

Comment:
I think it's safer to iterate over the result of `.collect()` just in case the internals change over time and the method returns more than one entry (or none!) in the future. Basically just use the same code as in `ThreadsWrapper`. 

Maybe extract the method somewhere to avoid duplication.